### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by Wend1go**
**Changes proposed in this pull request:**
- Add German translation for particle effects
- Add some missing translation strings

**Note:**
This is a WIP. Please don't merge this yet since I wasn't able to check in the IDE if the translated strings are appropriate due to #171

### CODE REVIEW

**Comment by CosmoMyzrailGorynych** [comment]
#171 is fixed, awaiting your review :+1:

**Comment by CosmoMyzrailGorynych** [comment]
This range is not needed now and should be deleted; it is now replaced by `debuggerToolbar` namespace:
https://github.com/ct-js/ct-js/blob/33eb4ba18e20850bb74b66c3fb39d77bc6aa145d/app/data/i18n/German.json#L515-L521

File: app/data/i18n/German.json [hunk 1]
```diff
@@ -399,6 +516,7 @@
         "reload": "Neu laden",
         "roomRestart": "Raum neu starten",
         "openExternal": "Im Browser öffnen",
-        "getQR": "QR Codes und lokale Server URL"
+        "getQR": "QR Codes und lokale Server URL",
+        "flipLayout": "Debugger Layout ändern"
```
Comment (CosmoMyzrailGorynych) [inline app/data/i18n/German.json]: The namespace `preview` is not needed now and should be deleted; it is now replaced by `debuggerToolbar` namespace.

**Review by CosmoMyzrailGorynych — CHANGES_REQUESTED** [review]

File: app/data/i18n/German.json (hunk 1 above)
Comment (CosmoMyzrailGorynych) [reply app/data/i18n/German.json]: Oh, they're in English.json as well. Whoopsie!

Comment (CosmoMyzrailGorynych) [reply app/data/i18n/German.json]: So I wrote an i18n linter and cleaned excess fields in all language files. A merge from `develop` to your branch — and it is resolved.

**Comment by Wend1go** [comment]
Ok, ready to merge. 😃

**Review by CosmoMyzrailGorynych — APPROVED** [review]

**Pull request merged**